#goal definition
string autonomy_launchfile
---
#result definition
---
#feedback